packages/manager/apps/public-cloud/src/index.html

```diff
@@ -38,7 +38,14 @@ <h1 class="mt-3 mb-5" data-ng-if="('preload_manager_ovh' | translate) !== 'prelo
                 </div>
             </div>
             <div data-otrs-popup></div>
-            <ovh-chatbot></ovh-chatbot>
+            <ovh-chatbot
```

Comment:
You remove the import but still use it ? Maybe I'm missing something :)

Reply:
:sparkles: It's magic. :sparkles: